src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -578,19 +707,16 @@ sub-<label>/[ses-<label>/]
     fmap/
         sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_epi.nii[.gz]
         sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_epi.json
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.nii[.gz]
+        sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.json
 ```
 
-The phase-encoding polarity (PEpolar) technique combines two or more Spin Echo
-EPI scans with different phase encoding directions to estimate the underlying
-inhomogeneity/deformation map. Examples of tools using this kind of images are
-FSL TOPUP, AFNI 3dqwarp and SPM. In such a case, the phase encoding direction is
-specified in the corresponding JSON file as one of: `i`, `j`, `k`, `i-`, `j-`,
-`k-`. For these differentially phase encoded sequences, one also needs to
-specify the Total Readout Time defined as the time (in seconds) from the center
-of the first echo to the center of the last echo (aka "FSL definition" - see
-[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and
-[here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For
-example
+The phase-encoding polarity (PEpolar) technique combines two or more EPI scans, with different phase encoding directions to estimate the underlying inhomogeneity/deformation 
+map. In the case of arterial spin labeling, this usually concerns two ‘m0scans’. Examples of tools using this kind of images are FSL TOPUP, AFNI 3dqwarp and SPM. In such a case, 
+the phase encoding direction is specified in the corresponding JSON file as one of: `i`, `j`, `k`, `i-`, `j-`,`k-`. For these differentially phase encoded sequences, one also 
+needs to specify the Total Readout Time defined as the time (in seconds) from the center of the first echo to the center of the last echo (aka "FSL definition" - see
+[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F)
+and [here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For example
```

Comment:
I think this only added one sentence to the entire thing, but the reformatting makes it hard to be sure. Can you only modify lines that must be modified?

This will need to be updated with the latest changes in `master`.

Reply:
Thanks, there were several existing fields (e.g. FlipAngle, EchoTime, RepetitionTime, SliceTime) where we needed to add an option/case for ASL. The BIDS government committee advised us to copy these fields fully and adapt them, in the ASL section. We perhaps have to discuss these. For this case: @patsycle can you only change the parts that we adapted, and keep the original formatting, etc such that it is easier to read the difference?

Reply:
@effigies indeed, we only added the ASL part, which is the sentence 'In the case of arterial .....two 'm0scans'. How should I change this practically?

Reply:
Practically? Copy that sentence into your clipboard or somewhere convenient, merge `master` in and just accept `master`'s version of this chunk of text. Then, find where you'd put the sentence in the new version of the text as a separate commit.

Reply:
@effigies Thanks. Just to be sure - this will probably merge all the other commits in other files from master to my branch - that is OK with you, right? If yes, then I will rebase master into my branch to keep my commits together.